EffectPackages.ini

```diff
@@ -296,3 +296,12 @@ DownloadUrl=https://github.com/Zenteon/ZN_FX/archive/refs/tags/v0.1.1.zip
 RepositoryUrl=https://github.com/Zenteon/ZN_FX
 EffectFiles=ZN_DAMP_RT.fx,ZN_GI.fx,ZN_Stylize.fx,ZN_WF.fx
 
+[32]
+PackageName=GShade-Shaders by Marot
+PackageDescription=A hodgepodge collection of original and legacy shaders maintained on behalf of the GShade project.
+InstallPath=.\reshade-shaders\Shaders\GShade-Shaders
+TextureInstallPath=.\reshade-shaders\Textures
+DownloadUrl=https://github.com/Mortalitas/GShade-Shaders/archive/refs/heads/master.zip
+RepositoryUrl=https://github.com/Mortalitas/GShade-Shaders
+EffectFiles=4xBRZ.fx,AdaptiveColorGrading.fx,Bumpmapping.fx,CA.fx,CathodeRayTube.fx,Censor.fx,Censor.fxh,CEOG.fx,ChromaSubSampling.fx,Clarity2.fx,ColorInversion.fx,ColorMod.fx,ColorToAlpha.fx,ColShift.fx,Composition.fx,CrossProcess.fx,CRTGeomMOD.fx,DepthSharpen.fx,DepthSharpenStaticDof.fx,DisplayMod.fx,Emboss.fx,ExtendedLevels.fx,FastSharpen.fx,Fisheye Horizontal.fx,Fisheye Vertical.fx,GAUSSIAN.fx,GaussianBloom.fx,GlobalAlpha.fx,GuestCRT.fx,Halation.fx,HueFX.fx,ImageSharpening.fx,Interlacing.fx,LevelIO.fx,LightPersistance.fx,LocalContrast.fx,LumaLines.fx,MotionFocus.fx,OrtonBloom.fx,PatternShading.fx,Pirate_Bloom.fx,Pirate_Depth_DOF.fx,Pirate_Depth_GI.fx,Pirate_FXAA.fx,PPFX_Bloom.fx,PPFX_Godrays.fx,PPFX_SSDO.fx,Reinhard.fx,RetroTV.fx,RetroTV.fxh,SimpleBloom.fx,SkySave.fx,SmartDeNoise.fx,SmartNoise.fx,Spotlight.fx,Spotlight.fxh,StageDepth.fx,StageDepth.fxh,SurfaceBlur.fx,ThreeColorGradient.fx,VerticalPreviewer.fx,VHSPro.fx,WatchDogs.fx,WinUaeMaskGlow.fx
```

Comment:
The `*.fxh` files aren't needed here, the setup tool will always keep all `.fxh` files, so should only list `*.fx` files (since this list directly affects what can be selected during the effect selection screen of the setup tool).

Reply:
Should be all set, thanks for catching that.